src/lib.rs

```diff
@@ -95,9 +103,91 @@ pub mod g_rpc {
     }
 }
 
+// A collection of functions for client side connections to the proxy server
 impl NetworkConfig {
     pub fn load(path: &str) -> Result<NetworkConfig, Box<dyn Error>> {
         let file = std::io::BufReader::new(File::open(path)?);
         Ok(serde_json::from_reader(file)?)
     }
+
+    /// get_client is an internal function to obtain the uds endpoint
+    ///
+    /// # Arguments
+    ///
+    /// * `p`: path to uds
+    ///
+    /// returns: Result<NetavarkProxyClient<Channel>, Status>
+    ///
+    /// # Examples
+    ///
+    /// ```
+    ///
+    /// ```
+    async fn get_client(p: String) -> Result<NetavarkProxyClient<Channel>, Status> {
+        let endpoint = match Endpoint::try_from("http://[::1]:10000") {
```

Comment:
What address is this? Do we listen on tcp?

Reply:
yeah, this is some suckage from tonic.  Nether @Jackbaude or I were able to exactly figure out how to get around this but basically the tonic maintainer stated this is part of the API.

Reply:
ack, please add a comment about this